langchain/src/chains/chat_vector_db_chain.ts

```diff
@@ -179,7 +175,30 @@ export class ChatVectorDBQAChain
     };
   }
 
-  static fromLLM(llm: BaseLLM, vectorstore: VectorStore): ChatVectorDBQAChain {
+  static getDefaultTemplates() {
```

Comment:
FYI @CarlosZiegler removing this part to remain consistent with the other chains